runtime/vm/vmprops.c

```diff
@@ -1720,13 +1720,13 @@ getPlatformPropertyList(JNIEnv *env, const char *strings[], int propIndex)
 		convertToUTF8(PORTLIB, unicodeTemp, userdir, EsMaxPath);
 		strings[propIndex++] = userdir;
 	}
-#endif /* JAVA_SPEC_VERSION < 17 */
+#endif /* JAVA_SPEC_VERSION < 11 */
 
-	if (JAVA_SPEC_VERSION < 12) {
-		/* Get the timezone */
-		strings[propIndex++] = "user.timezone";
-		strings[propIndex++] = "";
-	}
+#if JAVA_SPEC_VERSION == 8
```

Comment:
Why `JAVA_SPEC_VERSION == 8` here instead of grouping this above before the end of the `JAVA_SPEC_VERSION < 11` block?